source/FileSystemDsc.psd1

```diff
@@ -1,48 +1,41 @@
 @{
+    RootModule           = 'FileSystemDsc.psm1'
+
     # Version number of this module.
-    moduleVersion      = '0.0.1'
+    ModuleVersion        = '0.0.1'
 
     # ID used to uniquely identify this module
-    GUID               = '86a20a80-3bcd-477e-9b90-ec8d52fbe415'
+    GUID                 = '86a20a80-3bcd-477e-9b90-ec8d52fbe415'
+
+    CompatiblePSEditions = @('Core', 'Desktop')
 
     # Author of this module
-    Author             = 'DSC Community'
+    Author               = 'DSC Community'
 
     # Company or vendor of this module
-    CompanyName        = 'DSC Community'
+    CompanyName          = 'DSC Community'
 
     # Copyright statement for this module
-    Copyright          = 'Copyright the DSC Community contributors. All rights reserved.'
+    Copyright            = 'Copyright the DSC Community contributors. All rights reserved.'
 
     # Description of the functionality provided by this module
-    Description        = 'This module contains DSC resources for managing file systems.'
+    Description          = 'This module contains DSC resources for managing file systems.'
 
     # Minimum version of the Windows PowerShell engine required by this module
-    PowerShellVersion  = '5.1'
+    PowerShellVersion    = '5.1'
 
```

Comment:
During my testing I've found that also `RootModule` must be defined in the module manifest. Otherwise the class based resources are not found.

```
RootModule           = 'FileSystemDsc.psm1'
```